clarity/src/vm/database/clarity_db.rs

```diff
@@ -1777,8 +1792,8 @@ impl<'a> ClarityDatabase<'a> {
             stx_balance.amount_locked(),
             stx_balance.unlock_height(),
             cur_burn_height,
-            stx_balance.get_available_balance_at_burn_block(cur_burn_height, self.get_v1_unlock_height()),
-            stx_balance.has_unlockable_tokens_at_burn_block(cur_burn_height, self.get_v1_unlock_height()));
+            stx_balance.get_available_balance_at_burn_block(cur_burn_height, self.get_v1_unlock_height(), self.get_v2_unlock_height()),
```

Comment:
Hey, this doesn't need to be done right now, but this interface is going to get really clunky when there are multiple PoX unlock heights (thinking about the one that adds `.pox-3`).  We might want to think about passing a trait implementation here that just provides an API for getting all the block heights, so we don't have all these integers to pass around.

Reply:
Yep, I'll see about updating this in Epoch 2.3. I don't think that a new unlock height will need to be added in Epoch 2.3 (because everyone is already unlocked), so it may not get clunkier until future PoX versions. That could also be done as a non-consensus changing refactoring PR (that may be my preference).